spec/src/main/asciidoc/metrics.asciidoc

```diff
@@ -124,7 +124,7 @@ a| * `method` - the fully qualified method name
 | Name | `ft.circuitbreaker.calls.total`
 
 | Type in MP Metrics | `Counter`
-| Type in MP Telemetry | `LongCounter`
+| Type in MP Telemetry | `A counter that emits long`
```

Comment:
Where this field is no longer a class name, it should not be in backticks any more.